internal/controllers/clusterclass/clusterclass_controller.go

```diff
@@ -220,6 +278,20 @@ func reconcileConditions(clusterClass *clusterv1.ClusterClass, outdatedRefs map[
 	)
 }
 
+func statusVariableFromClusterClassVariable(variable clusterv1.ClusterClassVariable, from string) clusterv1.ClusterClassStatusVariable {
+	return clusterv1.ClusterClassStatusVariable{
+		Name: variable.Name,
+		// TODO: In a future iteration this should be false where definitions are equal.
+		DefinitionsConflict: true,
```

Comment:
Q: should this be false, given that we are failing if there is two variable definitions with the same name? (so there will be never a definition conflict)